string topicName
string destination
---
uint8 status